server/index.ts

```diff
@@ -111,26 +115,34 @@ const spectator = new Spectator(
     votedTetroToSpawn
 );
 const scene = new SceneTracker();
+const boardSync = new BoardSync(updateBoard);
 
 io.on("connection", (socket) => {
     scoreboard.initSocketListeners(socket, level);
     spectator.initSocketListeners(socket);
     queue.initSocketListeners(socket);
     scene.initSocketListeners(socket, scoreboard.finalScores);
     level.initSocketListeners(socket);
+    boardSync.initSocketListeners(socket);
 
     if (process.env.VITE_DISABLE_WAITING_ROOM) {
         socket.emit("initPlayer", playerCounter);
         playerCounter += 1;
         playerCounter %= 4;
     }
 
-    socket.on("playerMove", (...args) => {
-        socket.broadcast.emit("playerMove", ...args);
+    socket.on("playerMove", (playerId, state) => {
+        if (playerId === null || playerId === undefined) {
+            return;
+        }
+        socket.broadcast.emit("playerMove", playerId, state);
     });
-    socket.on("playerPlace", (...args) => {
-        console.log("player ", args[0], " placed.");
-        socket.broadcast.emit("playerPlace", ...args);
+    socket.on("playerPlace", (playerId, state) => {
+        if (playerId === null || playerId === undefined) {
```

Comment:
Just to be consistent, should these be replaced with a single `== null` as well?